src/middlewared/middlewared/client/client.py

```diff
@@ -141,7 +141,7 @@ def _unregister_call(self, call):
         self._calls.pop(call.id, None)
 
     def call(self, method, *params, **kwargs):
-        timeout = kwargs.pop('timeout', 30)
+        timeout = kwargs.pop('timeout', 60)
```

Comment:
Not sure how I feel about increasing overall default timeout, since it may degrade user experience if something gets really stuck for other things.
What do you think about using the timeout kwargs in the client calls you think would take longer?